src/assets/css/style.css

```diff
@@ -506,4 +506,17 @@ footer #version {
 
 .recap-filing-button > img{
   padding-top: 2px;
+}
+
+.recap-banner-inner-div{
```

Comment:
Classes shouldn't normally be named after elements. Maybe use a better rule to get these to apply like you want?

Reply:
got it! I've change the name of this class.